addons/portal_rating/controllers/portal_chatter.py

```diff
@@ -35,6 +35,13 @@ def portal_chatter_init(self, res_model, res_id, domain=False, limit=False, **kw
         result.update(self._portal_rating_stats(res_model, res_id, **kwargs))
         return result
 
+    def _get_void_portal_messages_domain(self):
+        """Override the domain to include messages with ratings."""
+        domain = super()._get_void_portal_messages_domain()
+        if request.env.context.get("rating_include"):
+            return expression.OR([domain, [("rating_ids", "!=", False)]])
```

Comment:
Rating ids being internal user only, are we sure portal won't ever use it ?

Reply:
What do you mean by the field is removed?
I just checked the SQL request and the field is still in the request:
![image](https://github.com/user-attachments/assets/966b989c-ee53-40d2-a543-e9c459b3a213)